test/01LessonOneTest.js

```diff
@@ -4,34 +4,66 @@ var assert = require('assert');
 
 
 function returnTrue() {
+  return true;
     // should return true
-
 }
 
 function returnFalse() {
+  return false;
     // should return false
-
 }
 
 function greaterThan(big, small) {
+  if(big>small){
+    return true;
+  }
+    else {
+      return false;
+    }
+    var big = 10;
+    var small = 0;
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
 
 }
 
 function lessThan(small, big) {
+  if(small<big){
+    return true;
+  }
+    else {
+      return false;
+    }
+    var big = 10;
+    var small = 0;
     // should return true if the first argument is less than the second argument
     // and return false if the first argument is greater than the second argument
 
 }
 
 function equalTo(a, b) {
+    if (a===b){
+      return true;
+    }
+    else{
+      return false;
+    }
+    var a=1;
+    var b=1;
```

Comment:
**a** and **b** are passed in as parameters, and you're redefining them as variables here.

This seems to be a pattern in your code, and this can cause problems. Redefining them here will cause the parameter values to be lost.

Hit me up on slack, and let's talk about what's going on here in this code.